### DISCUSSION TITLE
Music videos: Support for multiple <artist> tags

### DISCUSSION THREAD

**Discussion by rainerhillebrand** [Komet/MediaElch#1706] [Ideas]
The table in the article https://kodi.wiki/view/NFO_files/Music_videos in section "Music Video Tags" defines that multiple <artist> tags are supported in nfo files for Music Videos.
MediaElch seems to support only a single <artist> tag so that multiple artists can only be given in a single string.
When showing all artists of all Music Videos in Kodi (at least in Nexus 20.3) then you cannot select one of these multiple artists. Additional information about each artist cannot be shown via the music video item's context menu.
Wouldn't it make sense that MediaElch supports multiple <artist> tags for Music Videos nfo files as well?

**Comment by bugwelle** [comment]
Hi, _technically_, we don't have a musicvideo section. We do have concerts as well as artist/albums. But I agree that MediaElch should support more fields with multiple values. It's not just artist, but other fields as well.

Issue in https://github.com/Komet/MediaElch/issues/1709
